content/en/docs/reference/glossary/api-resource.md

```diff
@@ -0,0 +1,17 @@
+---
+title: API resource
+id: api-resource
+date: 2025-02-09
+full_link: /docs/reference/using-api/api-concepts/#standard-api-terminology
+short_description: >
+  A Kubernetes entity, representing an endpoint on the Kubernetes API server.
+
+aka:
```

Comment:
Does this work?
```suggestion
aka:
 - Resource
```

Reply:
good suggestion :) (lack of familiarity with the doc setup here, so advice like this is great)